aten/src/ATen/native/SummaryOps.cpp

```diff
@@ -65,4 +134,112 @@ _bincount_cpu(const Tensor& self, const Tensor& weights, int64_t minlength) {
   });
 }
 
+
+std::tuple<Tensor,Tensor> _histogram_cpu(
```

Comment:
I'm still reading through, but should there be a version of this histogram where weights is optional?

Reply:
Because of the way dispatch is done I think this needs a comment and a better name. Maybe `_histogram_uniform_binds_cpu`?

Reply:
Done.